src/diff.ts

```diff
@@ -16,21 +16,38 @@ export type Hunk = {
   startLine: number;
   endLine: number;
   diff: string;
+  commentChains?: {
+    comments: ReviewComment[];
+  }[];
 };
 
 export type FileDiff = File & {
   hunks: Hunk[];
 };
 
-export function parseFileDiff(file: File): FileDiff {
+type ReviewComment = {
+  path: string;
+  body: string;
+  line?: number;
+  in_reply_to_id?: number;
+  id: number;
+  start_line?: number | null;
+  user: {
+    login: string;
+  };
+};
```

Comment:
The type definition could be more precise about nullable fields. Consider using union types with `null` explicitly where applicable, and making required fields non-optional. For example:
```typescript
type ReviewComment = {
  path: string;
  body: string;
  line: number | null;
  in_reply_to_id: number | null;
  id: number;
  start_line: number | null;
  user: {
    login: string;
  };
};
```
This makes it clearer which fields can actually be null versus undefined.